portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['I have lived in the same house for my entire life!', 'I cage-dived with great white sharks.', 'Hablo español.', 'I used to perform on the flying trapeze.'];
```

Comment:
Split this line up so that everything fits on the screen without having to scroll horizontally.
```suggestion
      ['I have lived in the same house for my entire life!', 'I cage-dived with great white sharks.', 
      'Hablo español.', 'I used to perform on the flying trapeze.'];
```